task/task.json

```diff
@@ -119,6 +119,15 @@
             "groupName": "advanced",
             "required": false,
             "helpMarkDown": "The variable value which will be replaced by an empty string."
+        },
+        {
+            "name": "variablePrefix",
+            "type": "string",
+            "label": "Variable Prefix",
+            "defaultValue": "",
+            "groupName": "advanced",
+            "required": false,
+            "helpMarkDown": "The prefix that will be prepended to the variable name."
```

Comment:
I'd put more info on how it works: "The common variable prefix that will be prepended to the variable name. So if the prefix is "Prefix." and variable name is "Variable" than the search is made for "Prefix.Variable" and if it's not found than we make a search for "Variable" itself."